server/routes/journeys/referral/contributory-factor-comment/tests.cy.ts

```diff
@@ -0,0 +1,86 @@
+import { v4 as uuidV4 } from 'uuid'
+import { checkAxeAccessibility } from '../../../../../integration_tests/support/accessibilityViolations'
+import { injectJourneyDataAndReload } from '../../../../../integration_tests/utils/e2eTestUtils'
+
+context('test /referral/contributory-factor-comment', () => {
+  const uuid = uuidV4()
+  const START_URL = `${uuid}/prisoners/A1111AA/referral/start`
+  const PAGE_URL = `${uuid}/referral/code1-comment`
+
+  const getContinueButton = () => cy.findByRole('button', { name: /Continue/ })
+  const getComment = () => cy.findByRole('textbox', { name: 'Add a comment on type factors (optional)' })
+
+  const resetInputs = () => {
+    getComment().clear()
+  }
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubComponents')
+    cy.task('stubContribFactors')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubGetPrisoner')
+  })
+
+  it('should try out all cases', () => {
+    cy.signIn()
+    cy.visit(START_URL, { failOnStatusCode: false })
+
+    injectJourneyDataAndReload(uuid, {
+      referral: {
+        contributoryFactors: [
+          {
+            factorType: { code: 'CODE1', description: 'Type' },
+          },
+        ],
+      },
+    })
+
+    cy.visit(PAGE_URL)
+
+    checkAxeAccessibility()
+    validatePageContents()
+
+    validateErrorMessagesTextInputTooLong()
+
+    completeInputs()
+
+    getContinueButton().click()
+    cy.url().should('to.match', /\/contributory-factors-comments$/)
+    cy.go('back')
+
```

Comment:
reload after go back to ensure it's coming from express app, not cache from browser